Contributions/Using_Linux_in_a_Safe_System.md

```diff
@@ -0,0 +1,348 @@
+safety is certainly not one of them - nor should it be.
+
+Linux is neither conceived, nor evolved, to be safe. To be fair, it has
+never made any claim to be safe, therefore it should not be expected to be safe.
+Safety is not something that happens by accident. These considerations are
+intended to help with designing a system that is safe **despite** using Linux.
+
+All of the background justification, related to hardware (ARM64), Linux kernel
+architecture, and allocation of safety requirements, can be found in the documents
+mentioned in the [References](#References) section.
+
+In particular, the present document can be seen as a counterpart to the document
+**CheckList for Safety Claims on a Generic Linux System**.
+While the Checklist is meant to help with detecting problems, the present document
+wants to help with avoiding problems, upfront.
+
+The same disclaimer made for the checklist is valid here too: this document is
+not meant to provide full solutions to the problems listed.
+Solutions are often heavily contextualised, and must be designed to suit specific
+needs, in terms of use cases, safety requirements and system design.
+
+The evaluation of the very same mitigation strategy can range from overkill to
+insufficient, depending on where/how it is applied.
+
+
+## **Structure of the document**
+- The document starts with a section containing design considerations.
+- The following section contains several examples of how to apply such
+  considerations to the design of the Linux-bearing system.
+
+
+## **Considerations**
+
+**Premise:** some of the following considerations assume a fairly high level
+of freedom in designing the overall system.
+While one must acknowledge that in practice this freedom will not be always
+available, due to design constraints, it is equally true that attempting to
+cope with these constraints can lead to an even higher total cost of ownership.
+
+* **Clearly define requirements toward system integrity and availability, upfront**
+
+  Having to simply report failures (within a maximum allotted time - and how much?)
+  usually is significantly simpler than having to guarantee that a certain
+  functionality will not be interrupted.
+
+  Clarifying the scope of the design goes a long way in defining criteria for
+  identifying acceptable solutions.
+
+* **Linux is intrinsically not safe**
+
+  While Linux has many remarkable properties, safety is not one of them, which means
+  that it won't be possible to just allocate safety requirements to Linux, and expect
+  them to be automatically satisfied. Quite the opposite, actually.
+
+  Having to cope with safety requirements assigned to Linux will involve a price to
+  be paid, in one form of another.
+
+  The problem with said cost is that it might *also* affect negatively the goals
+  that led to the selection of Linux, in the first place. For example, if Linux was
+  selected because of its a rapid turnaround in fixing security bugs, selecting
```

Comment:
of its a rapid turnaround in fixing security bugs
--------------------
of its rapid turnaround in fixing security bugs